ioctl/Makefile

```diff
@@ -0,0 +1,5 @@
+NAME := test-ioctl
```

Comment:
note: there is a shortcut for adding unity tests, eg.
```make
$(eval $(call add_unity_test, test-unity))
```

for this the source file name and expected binary name must match (which is a good practice!)